### DISCUSSION TITLE
Setup screen in the beatmap editor not having buttons to reset the values like AR, OD, etc.

### DISCUSSION THREAD

**Discussion by Tenexxt** [ppy/osu#30270] [Q&A] [answered]
The title already explains this, the reason for this not being an issue is because im not sure if this was intentional or not.

The default values are easy to remeber and you most likely won't even need to reset them as you can just find yourself a new value instead of having to reset it first. Thats what i thought, but having the reset button just feels better, even if i don't use it idk. Not having it feels very weird to me so i was wondering if its me being weird or are others also put off by this.

_If you don't know what the reset button is, its this lil thingy. 
![{F5530BCC-F714-4758-84FF-E92FF78996DD}](https://github.com/user-attachments/assets/393ff2ad-be3f-4e80-8948-d5b2e7d3ca8e)
Its used to reset values... basically everywhere in the game._

**Comment by bdach** [comment, marked as answer]
Removed intentionally because the defaults in the context of the setup screen felt arbitrary (and there already was some arguing on what the defaults should *be*).
